composer.json

```diff
@@ -17,28 +17,29 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "ext-json": "*",
-        "graylog2/gelf-php": "^1.4.2",
-        "monolog/monolog": "^1.24 || ^2.0",
-        "sentry/sdk": "^3.1",
-        "sentry/sentry": "^3.1",
-        "sentry/sentry-symfony": "^4.0",
-        "symfony/config": "^3.4|^4.0|^5.0|^6.0",
-        "symfony/dependency-injection": "^3.4|^4.0|^5.0|^6.0",
-        "symfony/expression-language": "^3.0 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^3.4.26|^4.2.7|^5.0|^6.0",
-        "symfony/http-kernel": "^3.4|^4.0|^5.0|^6.0",
-        "symfony/monolog-bundle": "^3.4|^5.0"
+        "graylog2/gelf-php": "^1.4.2 || ^2.0",
+        "sentry/sentry-symfony": "^5.3",
+        "symfony/config": "^4.4.20||^5.0.11||^6.0||^7.0",
+        "symfony/dependency-injection": "^4.4.20||^5.0.11||^6.0||^7.0",
+        "symfony/http-kernel": "^4.4.20||^5.0.11||^6.0||^7.0",
+        "monolog/monolog": "^2.0.1",
```

Comment:
Please keep same order, composer has `sort-packages` config
```
    "config": {
        "sort-packages": true,
```
now dependency is in wrong place and will shift in case some other dependency will be added properly with `composer require ...`. Changed order makes reviewing changes harder.